# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/EitherParser.h
# Commit: cb0e58368cd4
# Full Hash: cb0e58368cd4d8e1f79b193b7ccae9586cc2ab50
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2020-11-14 09:46:25
# Regressor Bug: 1675241
# File Overlap Count: 1
# Description:
#   Bug 1675241 - Part 10: Use TaggedParserAtomIndex in StencilModuleEntry. r=tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96917
# ==============================================================================

diff -r 73cc5fa0e0bd -r cb0e58368cd4 js/src/frontend/EitherParser.h
--- a/js/src/frontend/EitherParser.h	Sat Nov 14 00:03:20 2020 +0000
+++ b/js/src/frontend/EitherParser.h	Sat Nov 14 00:03:28 2020 +0000
@@ -154,6 +154,11 @@
     ParserSharedBase& base = parser.match(detail::ParserSharedBaseMatcher());
     return base.liftParserAtomToJSAtom(parserAtom);
   }
+
+  CompilationInfo& getCompilationInfo() {
+    ParserSharedBase& base = parser.match(detail::ParserSharedBaseMatcher());
+    return base.getCompilationInfo();
+  }
 };
 
 } /* namespace frontend */